api/notd/api/models_v1.py

```diff
@@ -201,3 +221,6 @@ class RetrieveCollectionRequest(BaseModel):
 
 class RetrieveCollectionResponse(BaseModel):
     collection: ApiCollection
+
+class RetrieveCollectionStatisticsResponse(BaseModel):
```

Comment:
you should always have a request as well, even though its not used, just to higlight that it is indeed empty.